jans-cedarling/cedarling/src/authz/entities/meta.rs

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
I'm concerned that the entity metadata is hard coded. Let's say a developer adds the claim `org_id` to the `Jans::Workload`schema, so it looks like this:

```
    entity Workload  = {
        client_id: String,
        iss: TrustedIssuer,
        org_id: String
    };
``` 

The entity mapper in the Cedarling should automatically pick up this new claim. Otherwise we can't make any meaningful policies about access tokens--they will only contain information about the client_id and issuer...

Reply:
will be resolved in the https://github.com/JanssenProject/jans/issues/9677